apps/jobs/models.py

```diff
@@ -85,6 +85,11 @@ class Submission(TimeStampedModel):
     input_file = models.FileField(
         upload_to=RandomFileName("submission_files/submission_{id}")
     )
+    submission_input_file = models.FileField(
+        upload_to=RandomFileName("submission_files/submission_{id}"),
```

Comment:
Is this right? @savish28  Why is the upload_to location same as that of input_file? Will only one of them be uploaded in any scenario?
cc: @Ram81

Reply:
Actually, input_file would contain the docker image URI information in code-based challenges and submission_input_file would contain the output of the job created the code. All the submission-related file data is stored in submission_{id}, like stderr/stdout/input_file so I think this would be the best location to store the file.

Reply:
Got it. Thanks